classes/class-gf-field-hpsach.php

```diff
@@ -0,0 +1,416 @@
+<?php
+
+if (!class_exists('GFForms')) {
+    die();
+}
+
+/**
+ * Class GF_Field_HPSACH
+ */
+class GF_Field_HPSach extends GF_Field {
+    /**ACH element names*/
+    const HPS_ACH_CHECK_HOLDER_FIELD_NAME     = 'ach_check_holder';
```

Comment:
How come the check holder field name isn't prefixed like the others? These should be normalized. Looking at the group, it seems like this would be a fitting pattern: `hps_ach_{field_name}` where `{field_name}` is one of `check_holder`, `account`, `routing`, `account_type`, `check_type`